File: src/main/java/com/curseclient/mixin/accessor/render/AccessorModelEnderCrystal.java
package com.curseclient.mixin.accessor.render;

import net.minecraft.client.model.ModelEnderCrystal;
import net.minecraft.client.model.ModelRenderer;
import org.spongepowered.asm.mixin.Mixin;
import org.spongepowered.asm.mixin.gen.Accessor;

@Mixin(ModelEnderCrystal.class)
public interface AccessorModelEnderCrystal {

    @Accessor("cube")
    ModelRenderer getCube();

    @Accessor("glass")
    ModelRenderer getGlass();
}

File: src/main/kotlin/com/curseclient/mixin/accessor/render/AccessorModelEnderCrystal.kt
package com.curseclient.mixin.accessor.render

import net.minecraft.client.model.ModelEnderCrystal
import net.minecraft.client.model.ModelRenderer
import org.spongepowered.asm.mixin.Mixin
import org.spongepowered.asm.mixin.gen.Accessor

@Mixin(ModelEnderCrystal::class)
interface AccessorModelEnderCrystal {
    @get:Accessor("cube")
    val cube : ModelRenderer

    @get:Accessor("glass")
    val glass : ModelRenderer
}